e2e/src/main/java/org/jetbrains/bsp/bazel/base/BazelBspTestScenarioStep.java

```diff
@@ -22,7 +22,7 @@ public boolean executeAndReturnResult() {
 
     return Try.run(testkitCall)
         .onSuccess(e -> LOGGER.info("Step \"{}\" executed correctly!", testName))
-        .onFailure(e -> LOGGER.error("Step \"{}\" execution failed! Exception: {}", testName, e))
+        .onFailure(e -> LOGGER.error("Step \"" + testName + "\" execution failed!", e))
```

Comment:
why?

Reply:
To use the overload that accepts `Exception` so that it can be printed properly with stacktrace. Currently it is just `toString()`.

Reply:
ohh, yea, but first placeholder still can be used
```
.onFailure(e -> LOGGER.error("Step \"{}\" execution failed!", testName, e))
```

Reply:
I didn't suspect someone will check the type of last argument, but it seems to be the case, so I will fix this :p